gammapy/catalog/fermi.py

```diff
@@ -42,6 +42,18 @@
 log = logging.getLogger(__name__)
 
 
+def get_nonentry_keys(d, keys):
+    vals = [str(d[_]).strip() for _ in keys]
+    return ", ".join([_ for _ in vals if _ not in ["", "--"]])
+
+
+def get_nonentry_key(key):
+    if key.strip() == "":
+        return "--"
```

Comment:
Why are you returning `--` and not `""` directly?

Reply:
that's what the MaskedConstant returns when type is not str